modules/core/src/test/java/org/apache/ignite/internal/processors/cache/transactions/TxRecoveryConcurrentTest.java

```diff
@@ -103,7 +105,7 @@ public void testRecoveryNotDeadLockOnNearAndPrimaryFail() throws Exception {
 
             final TransactionProxyImpl<?, ?> tx = (TransactionProxyImpl<?, ?>)primary.transactions().txStart();
 
-            cache.put(key, key);
+            cache.put(key, key + iter);
```

Comment:
why not just (key, iter) ?

Reply:
Hm, by no reason in fact.  May be to imphasize the fact that it's something that is changed on each iteration.